test_scripts/API/Restructuring_OnResetTimeout/004_HMI_did_not_respond_OnResetTimeout_without_resetPeriod.lua

```diff
@@ -0,0 +1,50 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0189-Restructuring-OnResetTimeout.md
+-- User story: TBD
+-- Use case: TBD
+--
+-- Requirement summary: TBD
+--
+-- Description:
+-- In case:
+-- 1) RPC is requested
+-- 2) Some time after receiving RPC request on HMI is passed
+-- 3) HMI sends BC.OnResetTimeout(without resetPeriod) to SDL
+-- 4) HMI does not send response in 10 seconds after receiving request
+-- SDL does:
+-- 1) Respond with GENERIC_ERROR resultCode to mobile app in 10 seconds after receiving request
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local common = require('test_scripts/API/Restructuring_OnResetTimeout/common_OnResetTimeout')
+local commonRC = require('test_scripts/RC/commonRC')
+
+--[[ Test Configuration ]]
+runner.testSettings.isSelfIncluded = false
+
+--[[ Local Variables ]]
+local paramsForRespFunction = {
+	notificationTime = 2000
+}
+
+--[[ Scenario ]]
+runner.Title("Preconditions")
+runner.Step("Clean environment", common.preconditions)
+runner.Step("Start SDL, HMI, connect Mobile, start Session", common.start)
+runner.Step("App_1 registration", common.registerAppWOPTU)
+runner.Step("App_2 registration", common.registerAppWOPTU, { 2 })
+runner.Step("App_1 activation", common.activateApp)
+runner.Step("Set RA mode: ASK_DRIVER", commonRC.defineRAMode, { true, "ASK_DRIVER" })
+runner.Step("Create InteractionChoiceSet", common.createInteractionChoiceSet)
+
+runner.Title("Test")
+for _, rpc in pairs(common.rpcs) do
+  runner.Step("Send " .. rpc , common[rpc],
+	{ 10000, 10000, common.withoutResponseWithOnResetTimeout, paramsForRespFunction, common.rpcResponse })
```

Comment:
first parameter must be 13000, because notification is sent in 2 seconds after HMI request and reset period will be 10000